modules/optimum/optimum/intel/openvino/modeling_ov_utils.py

```diff
@@ -51,20 +48,30 @@ def load_ov_model_from_pytorch(model, inputs=None):
     import io
 
     buf = io.BytesIO()
+
+    tokenizer = AutoTokenizer.from_pretrained(model.name_or_path)
```

Comment:
Having to initialize a tokenizer just to get model input names is not ideal but I did not find another reliable way to get these names.